### DISCUSSION TITLE
How to change the color of gitmoditify files and directories

### DISCUSSION THREAD

**Discussion by Sammorn** [nvim-neo-tree/neo-tree.nvim#1503] [General]
How to change the color of gitmoditify files and directories
![20240622001546](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/91642847/a5236e0d-b486-461e-9cc6-d94a8f4b0972)

**Comment by rajayonin** [comment]
You can modify the highlight colors through Vim's `highlight` command. I personally use it to change the color of hidden files.

With [lazy.nvim](https://lazy.folke.io/), I just put it in the [`init`](https://lazy.folke.io/spec#spec-setup) property:
```lua
{
    "nvim-neo-tree/neo-tree.nvim",
    ...
    init = function()
      -- change dotfiles highlight color (set it to normal)
      vim.cmd('highlight NeoTreeDotfile guifg=NeoTreeNormal');
      vim.cmd('highlight NeoTreeGitIgnored guifg=#626262');  -- used to link to NeoTreeDotfile, undo that
    end,
    ...
}
```

You can see a full list of highlights and more information about them in `:h neo-tree-highlights` (or [doc/neo-tree.txt](https://github.com/nvim-neo-tree/neo-tree.nvim/blob/main/doc/neo-tree.txt)). The one that you're looking for in this case is `NeoTreeGitModified`.

**Reply by rajayonin** [reply]
TIP: you can use `:highlight <group>` to see the current value. In this case, we can see that `NeoTreeGitModified` links to `TreeGitModified`, which is set to `#81a1c1` in my colorscheme:
```
:highlight NeoTreeGitModified
NeoTreeGitModified xxx links to TreeGitModified
:highlight TreeGitModified
TreeGitModified xxx guifg=#81a1c1
```

So you should directly modify `TreeGitModified`:
```lua
vim.cmd('highlight TreeGitModified guifg=<your color>');
```
